stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +309,10 @@ public void setUsage(@Usage String usage) {
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_RECEIVER, mReceiver);
         putStripeJsonModelMapIfNotNull(hashMap, FIELD_REDIRECT, mRedirect);
 
+        if (mType != null) {
```

Comment:
LMK if you notice any other needed fixes -- I'll avoid pushing while you're looking at the PR